serverless/email/unsubscribe/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
+	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	appaws "github.com/brave-intl/bat-go/libs/aws"
+	"github.com/brave-intl/bat-go/libs/logging"
+	"github.com/rs/zerolog"
+)
+
+var (
+	ctx, logger     = logging.SetupLoggerWithLevel(context.Background(), zerolog.InfoLevel)
+	dynamoTableName = aws.String("unsubscribe")
+)
+
+// handler takes the api gateway request and sends a templated email
+func handler(request events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
+	// setup base aws config
+	config, err := appaws.BaseAWSConfig(ctx, logger)
+	if err != nil {
+		// failed to get the base aws config
+		return events.APIGatewayProxyResponse{
+			StatusCode: http.StatusInternalServerError,
+			Body:       http.StatusText(http.StatusInternalServerError),
+		}, fmt.Errorf("failed to get the base aws config: %w", err)
+	}
+	// setup dynamodb client
+	dynamoClient := dynamodb.NewFromConfig(config)
```

Comment:
I imagine there will be reasonable overhead associated with establishing a new dynamo connection in each handle request.  Perhaps something to address in future iterations

Reply:
resolved here https://github.com/brave-intl/bat-go/pull/1593/commits/814bbf32307bb940831c328eef1defd39a8bd1a6